js/ui/text_area.js

```diff
@@ -123,6 +124,8 @@ var TextArea = TextBox.inherit({
     _createInput: function() {
         var $input = $("<textarea>");
```

Comment:
This value can be stored in `this._$input` the editor.js
We can remove find operator in this._input function and use the cached value